src/css/main.css

```
@@ -863,15 +863,15 @@ li.active .ic_motor {
 }
 
 .ic_welcome {
-/*    background-image: url(../images/icons/cf_icon_welcome_grey.svg);*/
+    background-image: url(../images/icons/cf_icon_welcome_grey.png);
```

Stylelint (17.4.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)